DevEdu.API/DevEdu.Business/Services/UserService.cs

```diff
@@ -15,17 +15,16 @@ public UserService(IUserRepository userRepository)
 
         public int AddUser(UserDto dto)
         {
-           var addedUserId = _userRepository.AddUser(dto);
+            AuthenticationService authenticationService = new AuthenticationService();
```

Comment:
2 problems are here:
1. each service must be injected if it is present in the DI-setup
2. it's very bad to inject one service into another. the better solution is to call authenticationService from the controller

Reply:
it's awesome